Convert to one sentence per line

maximiliankolb opened the pull request:
This PR converts the **Managing Hosts** guide to follow the one sentence per line idea.
See #326

Disclaimer: Tables were omitted and will be taken care of at a later stage.

spetrosi reviewed (changes requested):
guides/doc-Managing_Hosts/topics/Synchronizing_Templates.adoc has two sentences on line 48. Other than that everything looks good.

maximiliankolb commented:
> guides/doc-Managing_Hosts/topics/Synchronizing_Templates.adoc has two sentences on line 48. Other than that everything looks good.

This was initially excluded as it's part of a table. Fixed and rebased. I will have a closer look at tables etc. at a later stage.

spetrosi commented:
> > guides/doc-Managing_Hosts/topics/Synchronizing_Templates.adoc has two sentences on line 48. Other than that everything looks good.
> 
> This was initially excluded as it's part of a table. Fixed and rebased. I will have a closer look at tables etc. at a later stage.
Ah sorry I did not notice that it is part of a table. It renders fine with the new line you added.
